core-test/src/test/scala/org/bitcoins/core/crypto/ECPublicKeyTest.scala

```diff
@@ -38,15 +38,20 @@ class ECPublicKeyTest extends BitcoinSUnitTest {
   }
 
   it must "add keys correctly" in {
-    forAll(CryptoGenerators.publicKey, CryptoGenerators.privateKey) {
-      case (pubKey, privKey) =>
-        val sumKeyBytes = NativeSecp256k1.pubKeyTweakAdd(pubKey.bytes.toArray,
-                                                         privKey.bytes.toArray,
-                                                         true)
-        val sumKeyExpected = ECPublicKey.fromBytes(ByteVector(sumKeyBytes))
-        val sumKey = pubKey.add(privKey.publicKey)
-
-        assert(sumKey == sumKeyExpected)
+    if (Secp256k1Context.isEnabled) {
```

Comment:
why is this not tested on bouncy castle?

Reply:
This test essentially compares secp to bouncy castle, and if we don't have secp there isn't much of a point in doing this